packages/docs/docs/auth/methods/index.md

```diff
@@ -94,3 +94,27 @@ Device authentication is designed for true machine connectivity, for example con
 For machine connectivity, we encourage use of [ClientCredentials](/docs/auth/methods/client-credentials) with tightly scoped [access controls](/docs/access/access-policies), giving minimal access to the host at the edge.
 
 If OAuth2 client credentials based authentication is not an option due to device limitations, [Basic Authentication](https://www.medplum.com/docs/sdk/core.medplumclient.setbasicauth) can be used to connect to Medplum. In general, OAuth2 is preferred to basic authentication. [Consuming webhooks](/docs/bots/consuming-webhooks) is an example application where Basic Authentication makes sense.
+
+## Security Best Practices
+
+All three implementations types will have tokens or client credentials with system access, and edge systems should follow best practices.
+
+### Client Authentication
+
+- Local disk should be encrypted
```

Comment:
Mention password requirements, SSO and 2FA for credentials as best practice.

Reply:
Yeah -- this section feels risky.  The most important point here is that Medplum assumes that the user is using a HIPAA compliant device.  We should lean on HIPAA here.  We can list these as "for example" bullet points, but it should be clear that these are just examples, and that your compliance team / legal team / system administrator is responsible for HIPAA compliance and security.